test/data/test_functional.py

```diff
@@ -186,7 +186,7 @@ def test_load_and_save(self):
 
         with self.subTest('torchscript'):
             save_path = os.path.join(self.test_dir, 'regex_torchscript.pt')
-            tokenizer = regex_tokenizer(patterns_list).to_ivalue()
+            tokenizer = regex_tokenizer(patterns_list).__prepare_scriptable__()
```

Comment:
Same here

Reply:
See the comment above. We want to save a torchbind version model here, not exactly the JIT mode.